org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/format/DOMElementFormatter.java

```diff
@@ -76,7 +76,7 @@ private int formatStartTagElement(DOMElement element, XMLFormattingConstraints p
 		FormatElementCategory formatElementCategory = parentConstraints.getFormatElementCategory();
 		int startTagOffset = element.getStartTagOpenOffset();
 		boolean addLineSeparator = element.getParentElement() == null && element.getPreviousSibling() == null;

```

Comment:
Defines startTagCloseOffset variable here and rename startTagOffset  with startTagOpenOffset